tests/test_core/test_new_sequence.py

```diff
@@ -30,6 +36,172 @@ def test_moltype_make_bytes_seq():
 
 
 # Tests from test_sequence.py
+def test_to_json():
+    """to_json roundtrip recreates to_dict"""
+    r = new_moltype.DNA.make_seq(seq="AAGGCC", name="seq1")
+    got = json.loads(r.to_json())
+    seq = new_sequence.SeqView(seq="AAGGCC", seqid="seq1").to_rich_dict()
+
+    expect = {
+        "name": "seq1",
+        "seq": seq,
+        "moltype": r.moltype.label,
+        "info": None,
+        "type": get_object_provenance(r),
+        "version": __version__,
+        "annotation_offset": 0,
+    }
+
+    assert got == expect
+
+
+@pytest.mark.xfail(
+    reason="NotImplementedError: deserialising 'cogent3.core.new_sequence.DnaSequence' from json"
+)
+def test_offset_with_multiple_slices(DATA_DIR):
+    seq = new_moltype.DNA.make_seq(
+        seq="ACCCCGGAAAATTTTTTTTTAAGGGGGAAAAAAAAACCCCCCC", name="22"
+    )
+    gff3_path = DATA_DIR / "ensembl_sample.gff3"
+    seq.annotate_from_gff(gff3_path)
```

Comment:
we are discontinuing support for this method, so please add a comment to directly assign an annotation_db